### PR TITLE
174 auto environments

### PR DESCRIPTION
**Pull request by krzysztof-rodzaj-mobica**

### CODE REVIEW

File: tests/environment_windows_amd64_freeze.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - conda-forge
+
+dependencies:
+  # Define conda-forge packages here -> https://anaconda.org/search
+  # When available, prefer the conda-forge packages over pip as installations are more efficient.
+  - python=3.9.13               # https://pyreadiness.org/3.9/ 
+  - pip=22.1.2                  # https://pip.pypa.io/en/stable/news/
+  - pip:
+      # Define pip packages here -> https://pypi.org/
+      # - rpaframework==22.5.3    # https://rpaframework.org/releasenotes.html
```
Comment (Anna-Majda-Mobica) [inline tests/environment_windows_amd64_freeze.yaml]: Czy potrzebujemy tych kilka zakomentowanych linii kodu?

Comment (krzysztof-rodzaj-mobica) [reply tests/environment_windows_amd64_freeze.yaml]: Fixed

File: tests/linux_env.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo apt-get install xdotool
```
Comment (Mariusz-Poznanski-Mobica) [inline tests/linux_env.sh]: Additionally we need here:
rclone  
git 
p7zip-full

File: tests/environment_linux_amd64_freeze.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+dependencies:
+  # Define conda-forge packages here -> https://anaconda.org/search
+  # When available, prefer the conda-forge packages over pip as installations are more efficient.
+  - python=3.9.13               # https://pyreadiness.org/3.9/ 
+  - pip=22.1.2                  # https://pip.pypa.io/en/stable/news/
+  - pip:
+      # Define pip packages here -> https://pypi.org/
+      - robotframework==6.0.2
+      - selenium==4.8.2
```
Comment (Mariusz-Poznanski-Mobica) [inline tests/environment_linux_amd64_freeze.yaml:16]: I think selenium library for robotframework is enough.  I do not install it in docker container and it works

**Review by Anna-Majda-Mobica — APPROVED** [review]

**Comment by Anna-Majda-Mobica** [comment]
approving after entering changes required by Mariusz

File: tests/linux_env.sh (hunk 2 above)
Comment (krzysztof-rodzaj-mobica) [reply tests/linux_env.sh]: I assume git will be installed to clone our code, so it it unnecessary here. (other two are added)

File: tests/environment_linux_amd64_freeze.yaml (hunk 3 above)
Comment (krzysztof-rodzaj-mobica) [reply tests/environment_linux_amd64_freeze.yaml:16]: If I remember correctly selenium will be installed anyway for robotframework-selenium_library to work.

**Review by Mariusz-Poznanski-Mobica — APPROVED** [review]

File: tests/environment_linux_amd64_freeze.yaml (hunk 3 above)
Comment (krzysztof-rodzaj-mobica) [reply tests/environment_linux_amd64_freeze.yaml:16]: obraz

**Pull request merged**